### PR TITLE
Created Classes and wrote methods as described in the task

### PR DESCRIPTION
**Pull request by Jacalius**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            Ball randomBall = lottery.getRandomBall();
+            System.out.println(randomBall.toString());
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:9]:
```suggestion
            System.out.println(randomBall);
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: image

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+
+        Random random = new Random();
+        int ballNumber = random.nextInt(100);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:10]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_POSSIBLE_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = random.nextInt(MAXIMUM_POSSIBLE_NUMBER);
+        return new Ball(color.getRandomColor().toString(), ballNumber);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:12]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 5]
```diff
@@ -1,7 +1,12 @@
...
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```
Comment (MentorMA) [inline src/main/java/core/basesyntax/ColorSupplier.java:10]: Return color in String Type

File: src/main/java/core/basesyntax/Ball.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
```
Comment (MentorMA) [inline src/main/java/core/basesyntax/Ball.java:26]: If you not use you can clear this methods

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/ColorSupplier.java (hunk 5 above)
Comment (MAMentorFX) [reply src/main/java/core/basesyntax/ColorSupplier.java:10]: not fixed

File: src/main/java/core/basesyntax/Lottery.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_POSSIBLE_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = random.nextInt(MAXIMUM_POSSIBLE_NUMBER);
```
Comment (MAMentorFX) [inline src/main/java/core/basesyntax/Lottery.java:11]: random.nextInt() returns 0 inclusively and 100 exclusively

**Review by MAMentorFX — CHANGES_REQUESTED** [review]

**Review by MentorMA — APPROVED** [review]

**Pull request closed without merging**